apps/prairielearn/src/sprocs/assessment_instances_insert.sql

```diff
@@ -108,8 +109,8 @@ BEGIN
 
     -- ######################################################################
     -- Add a log entry for the original time limit of the assessment instance
-    INSERT INTO assessment_state_logs (open, assessment_instance_id, date_limit, auth_user_id)
-    VALUES (true, assessment_instance_id, date_limit, authn_user_id);
+    INSERT INTO assessment_state_logs (open, assessment_instance_id, date_limit, auth_user_id, client_fingerprint_id)
+    VALUES (true, assessment_instance_id, date_limit, authn_user_id, last_client_fingerprint_id);
```

Comment:
I don't think `last_client_fingerprint_id` is a column in the `assessment_state_logs` table?

Reply:
We might want to call the argument `client_fingerprint_id`; it's an implementation detail of this sproc that it ends up being passed to the `last_client_fingerprint_id` column.

Reply:
That makes sense. I have updated this to call the argument `client_fingerprint_id`.